src/components/Signup.js

```diff
@@ -0,0 +1,5 @@
+<template>
+    <div>
+        Sign up
+    </div>
```

Comment:
@adenali20  here please close the div tag

Reply:
@abdilatifbashir  Done